internal/service/ecr/pull_through_cache_rule.go

```diff
@@ -50,6 +51,14 @@ func ResourcePullThroughCacheRule() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
+			"credential_arn": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ValidateFunc: validation.StringMatch(
```

Comment:
There are multiple AWS partitions, recommend you simply validate that a ARN is provided, and let the service API do the rest.

```suggestion
				ValidateFunc: verify.ValidARN,
```

Reply:
Done ✅